### ISSUE TITLE
Unable to input text for edit text using Jetpack compose

### ISSUE THREAD

**Issue by stowy** [serhatbolsu/robotframework-appiumlibrary#410]
Not sure if this is a robot framework or an Appium issue, but trying to input text using:

```
Input Text    accessibility_id=email    ${VALID_EMAIL}
```

Fails when using jetpack compose to set accessibility IDs, eg using:

```
                    OutlineTextField(
                        value = email,
                        onValueChange = viewModel::onEmailChange,
                        keyboardOptions = KeyboardOptions(
                            keyboardType = KeyboardType.Email,
                            imeAction = ImeAction.Next
                        ),
                        modifier = Modifier
                            .fillMaxWidth()
                            .focusRequester(focusRequester)
                            .semantics(mergeDescendants = true) {
                                testTag = "email";
                                testTagsAsResourceId = true},
                    )
```

It says the element cannot have the value set, eg see attached screenshot

<img width="1711" alt="Screenshot 2023-12-04 at 4 55 04 pm" src="https://github.com/serhatbolsu/robotframework-appiumlibrary/assets/5924628/5d95b14d-2ca7-46d3-a3e6-b21bb7342182">

If you click on the edit text above and send it passes the text

**Comment by stowy** [comment]
I [duplicated this issue](https://github.com/appium/appium-uiautomator2-server/issues/577) over at Appium UIAutomator github and they suggested possibly using [mobile:type](https://github.com/appium/appium-uiautomator2-driver?tab=readme-ov-file#mobile-type) to type the text rather than send text. Does Robot framework support sending text via this method?

**Comment by emanlove** [comment]
Due to lack of activity I am closing this ticket.  If you feel this is still relevant please open a new one.

Note also that AppiumLibrary is currently going through some revisions to make it useful and compatible with current tools. If you want to see what we have planned review the milestones or reach out to the current maintainers.
